c/reducercommutativity/rangesum05_false-unreach-call.yml

```diff
@@ -1,9 +1,9 @@
-program_files:
+input_files:
   - rangesum05_false-unreach-call.i
```

Comment:
Can the file names be paths relative to the directory where the yml file is placed? This would be necessary for Java where a benchmark consists of an entire source tree.

Reply:
Am I assuming correctly that the yml file is passed to the tool wrapper and the tool wrapper has to parse the source files to process from the yml file?

Reply:
Path in files that are parsed by BenchExec are always relative to the directory where the parsed file is placed. Paths in `input_files` can also be directories.

BenchExec will parse the yml files, the tool-info modules won't have access to them.